src/presets/React.d.ts

```diff
@@ -22,7 +22,7 @@ declare module 'bem-react-core' {
         elemMods?: Mods;
     }
 
-    type Mix = JsonMix | string | JSX.Element;
+    type Mix = JsonMix | JsonMix[] | string | JSX.Element;
```

Comment:
I think it would be correct to add `JSX.Element[]` also

Reply:
done